package.json

```diff
@@ -59,12 +59,12 @@
     "markdown-toc": "^1.2.0",
     "mocha": "^9",
     "nyc": "^14.1.1",
-    "oclif": "^2.5.0",
+    "oclif": "github:Souvikns/oclif#main",
     "rimraf": "^3.0.2",
```

Comment:
it says https://github.com/asyncapi/cli/runs/5496164139?check_suite_focus=true#step:9:12 but we have `rimraf`